nbs/020_micro_passes.ipynb

```diff
@@ -54,7 +54,7 @@
     "    schema_match,is_of_schema,type_merge,schema_merge\n",
```

Comment:
<p>in this nb, all the graphical outputs disappeared. Maybe you didn't run it before pushing using nbdev_test or something.</p><p>Look how it looks like in nbreview.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spannerlib/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>